installer/tue-install-impl.bash

```diff
@@ -777,7 +835,16 @@ ROS_PACKAGE_INSTALL_DIR=$TUE_SYSTEM_DIR/src
 
 TUE_INSTALL_SCRIPTS_DIR=$TUE_DIR/installer
 
-TUE_INSTALL_STATE_DIR=/tmp/tue-installer/$stamp
+TUE_INSTALL_GENERAL_STATE_DIR=/tmp/tue-installer
+if [ ! -d $TUE_INSTALL_GENERAL_STATE_DIR ]
+then
+    tue-install-debug "mkdir $TUE_INSTALL_GENERAL_STATE_DIR"
+    mkdir "$TUE_INSTALL_GENERAL_STATE_DIR"
```

Comment:
You could use the `mktemp` utility to safely create a temporary directory with a prefix.